chat-widget/src/common/telemetry/loggers/ariaTelemetryLogger.ts

```diff
@@ -25,6 +27,20 @@ export const ariaTelemetryLogger = (ariaTelemetryKey: string,
             if (!isNullOrEmptyString(collectiorUriForTelemetry)) {
                 configuration.collectorUri = collectiorUriForTelemetry;
             }
+            else{
+                if(TelemetryManager.InternalTelemetryData.environmentVersion == EnvironmentVersion.prod){
+                    const orgUrl = TelemetryManager.InternalTelemetryData?.orgUrl;
+                    if(!isNullOrUndefined(orgUrl)){
+                        // If the given org is a Production EU org, modify the Aria collector uri
+                        const region = getDomain(orgUrl);
+
+                        if (!isNullOrUndefined(region) && region === AriaTelemetryConstants.EU) {
```

Comment:
"!isNullOrUndefined(region) && " is not necessary since you're using "===" for comparison later

Reply:
Fixed. Thanks for suggesting.